woocommerce/woocommerce
# Add back the code reference updates action

### Changes proposed in this Pull Request:

This adds back the workflow that triggers a Github Pages deploy of the [Code Reference repository](https://github.com/woocommerce/code-reference). This triggers when a release is published.

### How to test the changes in this Pull Request:

I have tested this in my own fork and I think thats probably enough for now. You can see the run here: https://github.com/samueljseay/woocommerce/actions/runs/3269328014/jobs/5376760973

What you'll notice is that the run failed, this is because of an issue in the `code-reference` repo, but triggering the workflow works as expected. The next step will be to diagnose issues with the `code-reference` repo.

### FOR PR REVIEWER ONLY:
